sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.catalyst.expressions
 
 import scala.collection.mutable
+import scala.jdk.CollectionConverters.CollectionHasAsScala
```

Comment:
why this import?

Reply:
Because it is necessary to use from `java(List)` to `scala(Iterable)`, as follows:
<img width="796" alt="image" src="https://github.com/user-attachments/assets/3999af76-17fe-4c52-a389-ae4759e8249d">